frontend/pages/hosts/details/HostDetailsPage/_styles.scss

```diff
@@ -72,6 +72,10 @@
         }
       }
 
+      &__disk-space-wrapper {
+        display: inline-block;
+      }
+
```

Comment:
This is needed for the larger tooltip hover area, without it, the hover area is only 4px in height >.<